### PR TITLE
docs(aiplatform): Add PredictCodeCompletionCommentSample #2391

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

File: aiplatform/api/AIPlatform.Samples.Tests/PredictCodeCompletionCommentTest.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
```
Comment (hemanshv) [inline aiplatform/api/AIPlatform.Samples.Tests/PredictCodeCompletionCommentTest.cs:1]: mostly the same comments as of https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2405

**Review by hemanshv — CHANGES_REQUESTED** [review]

File: aiplatform/api/AIPlatform.Samples.Tests/PredictCodeCompletionCommentTest.cs (hunk 1 above)
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples.Tests/PredictCodeCompletionCommentTest.cs:1]: @hemanshv In the latest commit, I changed to use First() instead. Let me know if anything else.

**Comment by meteatamel** [comment]
Added Amanda for her review

File: aiplatform/api/AIPlatform.Samples/PredictCodeCompletionCommentSample.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        var client = new PredictionServiceClientBuilder
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Learn how to create prompts to work with a code model to create code completion suggestions:
+        // https://cloud.google.com/vertex-ai/docs/generative-ai/code/code-completion-prompts
```
Comment (amanda-tarafa) [inline aiplatform/api/AIPlatform.Samples/PredictCodeCompletionCommentSample.cs:45]: Links to docs on this one. Just be consistent. And in general if explanations and links are included in the documentation where the sample is shown, we don't need to have these in code.

File: aiplatform/api/AIPlatform.Samples.Tests/PredictCodeCompletionCommentTest.cs (hunk 1 above)
Comment (hemanshv) [inline aiplatform/api/AIPlatform.Samples.Tests/PredictCodeCompletionCommentTest.cs:1]: place holder to undo request changes

File: aiplatform/api/AIPlatform.Samples/PredictCodeCompletionCommentSample.cs (hunk 2 above)
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/PredictCodeCompletionCommentSample.cs]: Right. I removed the comment.

**Pull request merged**